src/api.v2/controllers/cellLevelMetaController.js

```diff
@@ -44,7 +45,27 @@ const updateCellLevelMetadata = async (req, res) => {
   await new CellLevelMeta().updateById(cellMetadataFileId, snakeCasedKeysToPatch);
   res.json(OK());
 };
+
+const downloadCellLevelFile = async (req, res) => {
+  const { fileId, fileName } = req.params;
+  logger.log('Getting signed url to download cell level meta file');
+  const link = await new CellLevelMeta().getDownloadLink(fileId, fileName);
+
+  res.json(link);
+  logger.log('Got download link successfully');
+};
+
+const deleteMetadata = async (req, res) => {
+  const { experimentId } = req.params;
+  logger.log('Deleting cell level metadata for experiment ', experimentId);
+  await new CellLevelMetaToExperiment().delete({ experiment_id: experimentId });
+  logger.log('Deleted cell level metadata for experiment ', experimentId);
+  res.json(OK());
+};
+
 module.exports = {
   uploadCellLevelMetadata,
   updateCellLevelMetadata,
+  downloadCellLevelFile,
+  deleteMetadata,
```

Comment:
same comments apply as for downloadCellLevelFile <-- please make all naming consistent